### ISSUE TITLE
Incorrect indentation error

### ISSUE THREAD

**Issue by maxim123** [vianuevm/cppStyle#124]
This produces an indentation error:

```
if (condition)
{
    do something
}
else
{
    do something else
}
```

But this doesn't:

```
if (condition) {
    do something
}
else
{
    do something else
}
```

**Comment by vianuevm** [comment]
It looks like it has a bug when there's block bracing - looks like a pretty simple bug to fix.

**Comment by willwolfram18** [comment]
The issue SHOULD have been fixed in a14009fc9666e4f70036ed380337160e8be3508f
